### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by aliabbani**
## Blog app - creating a data model

In this milestone I:
- Built my project schema.
Created and run the necessary migration files.
Table and column names should match the ERD diagram.
note: photo for users table should be a link to an image
Foreign keys is be included.
All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20220114195219_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :postscounter
```
Comment (bimbolabuari) [inline db/migrate/20220114195219_create_users.rb:7]: As best practices, let's adhere to ruby/rails naming conventions. `:postscounter` should be in snake case i.e. `:posts_counter.` Kindly make this change. 👍

File: db/migrate/20220114195311_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.integer :commentscounter
+      t.integer :likescounter
```
Comment (bimbolabuari) [inline db/migrate/20220114195311_create_posts.rb:7]: Let's work with ruby/rails naming conventions here also. 🙂 
```
t.integer :commentscounter
t.integer :likescounter
```
should be
```
 t.integer :comments_counter
 t.integer :likes_counter 
```

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @aliabbani , 

You have done a good job so far 🚀  



There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: db/migrate/20220114195219_create_users.rb (hunk 1 above)
Comment (aliabbani) [reply db/migrate/20220114195219_create_users.rb:7]: Done. Thank you @bimbolabuari

File: db/migrate/20220114195311_create_posts.rb (hunk 2 above)
Comment (aliabbani) [reply db/migrate/20220114195311_create_posts.rb:7]: Done. Thank you @bimbolabuari

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @aliabbani, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**